docs/public/content/demo/demo_guide/index.md

```diff
@@ -8,7 +8,7 @@ weight = 1
 
 Demo can be accessed at: [https://irims-demo.msupply.org:50000/](https://irims-demo.msupply.org:50000/)
```

Comment:
![Screen Shot 2022-05-31 at 10 45 26 AM](https://user-images.githubusercontent.com/16461988/171065535-703416b9-1a53-4bb1-8dca-36d7948e47f6.png)

Suggestion: Update the server with latest after this : ) So we allow people checking it out with the proper name